src/components/Calculator.jsx

```diff
@@ -0,0 +1,145 @@
+import { useState } from "preact/hooks";
+
+export default function Calculator() {
+  const [developersCount, setDevelopersCount] = useState(5);
+  const [buildTimeSaved, setBuildTimeSaved] = useState(50);
+  const [averageBuildTime, setAverageBuildTime] = useState(10);
+  const [averageMonthlySalary, setAverageMonthlySalary] = useState(4000);
```

Comment:
Should we set the default higher here? I don't think 4k is an average monthly salary of an iOS developer 😄

Reply:
Depends on the country but yeah, let's lift it up.